grammar.js

```diff
@@ -334,12 +336,13 @@ module.exports = grammar({
 
       const hex_literal = seq(
         choice('0x', '0X'),
+		    optional('.'),
```

Comment:
Can you please add test for it? Just add an example here:

https://github.com/MunifTanjim/tree-sitter-lua/blob/0fc89962b7ff5c7d676b8592c1cbce1ceaa806fd/test/corpus/expressions.txt#L43-L60